utils/reporting/tfm/tfm-reporting.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * TFM REPORTING
+ * **************
+ * Purpose of this script is to generate a bespoke
+ * report of all completed deals from TFM
+ */
+
+const CONSTANTS = require('../../data-migration/constant');
+const { getCollection, disconnect } = require('../../data-migration/helpers/database');
+const { write } = require('../../data-migration/helpers/io');
+
+// ******************** DEALS *************************
+/**
+   * Return all the TFM deals with `MIA/MIN` filter.
+   * @param {Integer} EPOCH Fetch records greater than.
+   * Defaulted to `1648684800` (31-03-2022). This argument
+   * accepts EPOCH with `ms`
+   * @returns {Object} Collection object
+   */
+const getTfmDeals = (epoch = 1648684800) => getCollection(
+  CONSTANTS.DATABASE.TABLES.TFM_DEAL,
+  { 'tfm.stage': 'Confirmed', 'tfm.dateReceivedTimestamp': { $gte: epoch } },
+);
+
+// ******************** FORMATTING *************************
+
+/**
+    * Strips commas and return UKEF exposure
+    * as number.
+    * @param {String} string Raw string with commas
+    * @param {Boolean} Number Whether output should be a `Number`
+    * @returns {Integer} Formatted value
+    */
+const stripCommas = (string, number = false) => {
```

Comment:
I've seen we have another file called `rad-reporting.js`. This function exists in that file as well. Is it worth creating a helper function and loading these in? to avoid duplications?

Reply:
Excellent point @IgnatG have re-factored code.